### ISSUE TITLE
Changing rooms creates drawing offset in undocked viewers

### ISSUE THREAD

**Issue by artisteacher** [opentoonz/opentoonz#4311]
**Describe the bug**
When working with multiple monitors with differing resolutions, changing the room creates offset in undocked viewers/comboviewers.
See [the comment here](https://github.com/opentoonz/opentoonz/pull/4292#issuecomment-1061412137) and [here](https://github.com/opentoonz/opentoonz/pull/4292#issuecomment-1061717731) and [the video in the comment here](https://github.com/opentoonz/opentoonz/pull/4292#issuecomment-1059659067)

**To Reproduce**
List steps to reproduce the behavior.
1. Connect a screen tablet to your computer as an extended desktop
2. If it's not already, make the lower resolution device the main monitor
4. Open OT
5. Add an undocked combo viewer window and move it to the other screen 
6. Draw something
7. Change rooms
9.  Return to the combo viewer and try to draw

**Expected behavior**
Changing rooms shouldn't create offset in undocked viewers

**Comment by artisteacher** [comment]
As @manongjohn found, resizing the combo viewer fixes the offset until a room is changed again.

**Comment by DarrenTAnims** [comment]
Have you tried the latest nightly? There was a fix only a day or 2 ago for screens with different resolutions.

**Comment by artisteacher** [comment]
Yeah, I tested that fix and it doesn't help in this particular case.
